app/src/main/java/com/imashnake/animite/features/home/Home.kt

```diff
@@ -219,7 +237,7 @@ fun HomeRow(
     list: List<Media.Medium>,
     title: String,
     onItemClicked: (Media.Medium) -> Unit,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
```

Comment:
Trailing commas can be enforced using detekt in #57, can we keep this PR focused to fixing light mode (#98)?